src/build/experimental-browser.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+const resolveBin = require('resolve-bin')
+const execa = require('execa')
+const {fromAegir} = require('./../utils')
+const bin = resolveBin.sync('webpack-cli')
+
+module.exports = (argv) => {
+  const input = argv._.slice(1)
+  const useBuiltinConfig = !input.includes('--config')
+  const config = useBuiltinConfig
+    ? ['--config', fromAegir('src/config/webpack.config.js')]
+    : []
+  return execa(bin, [
+    ...config,
```

Comment:
Yeah, since https://github.com/ipfs/community/pull/313 we only support latest stable + LTS releases of node.js https://github.com/ipfs/community/blob/master/js-code-guidelines.md#supported-versions